pkg/oci/cache.go

```diff
@@ -133,10 +137,20 @@ func (cache *Cache) FetchReference(ref *Reference) (*CacheRefSummary, error) {
 			if err != nil {
 				return &r, err
 			}
+			configBytes, err := cache.fetchBlob(r.Config)
```

Comment:
I'm not understand how `Config` filed load and save model, the `Config` field looks like `Metadata` filed, why we need both of them simultaneously.

Reply:
Maybe we will use it in the future, thus just keep the raw data here.

Reply:
ok, LGTM